app/assets/stylesheets/components/_tabs.scss

```diff
@@ -0,0 +1,7 @@
+.tab { 
+  display: none;
+}
+
+.active {
```

Comment:
We can also define these:
```
.tab {
  display: none;

  &.active {
    display: block;
  }
}
```

Which will scope the `.active` behaviour to `.tab` elements only.
Question is: do we want to keep the `.active` more generic and use it elsewhere? or do we prefer the scoping?

Reply:
Yes, I think it's better to keep it more generic, I'm pretty sure I can use it also for trips :)

Reply:
I'm gonna to use the new JS for the trips in the next PR.